### ISSUE TITLE
System.TypeLoadException : Inheritance security rules violated by type: 'Apex.NuGetClient.TestServices.NuGetApexTestService'. Derived types must either match the security accessibility of the base type or be less accessible.

### ISSUE THREAD

**Issue by v-atian** [v-atian/NuGetClientTests#39]
[5/8/2018 12:11:21 AM Error] [xUnit.net 00:00:06.4778327]     AbuseLink - NetStandardClassLibrary.v_2_0.VS [FAIL]
[5/8/2018 12:11:21 AM Informational] [xUnit.net 00:00:06.4797796]       System.TypeLoadException : Inheritance security rules violated by type: 'Apex.NuGetClient.TestServices.NuGetApexTestService'. Derived types must either match the security accessibility of the base type or be less accessible.
[5/8/2018 12:11:21 AM Informational] [xUnit.net 00:00:06.4811454]       Stack Trace:
[5/8/2018 12:11:21 AM Informational] [xUnit.net 00:00:06.4825971]            at NuGetClient.Test.Integration.Apex.ApexBaseTestClass..ctor()
[5/8/2018 12:11:21 AM Informational] [xUnit.net 00:00:06.4828726]         test\NuGet.Tests.Apex\NuGet.Client.Tests\Apex\IntegrationNuGetTestClass.cs(32,0): at NuGetClient.Test.Integration.Apex.IntegrationNuGetTestClass..ctor(IContextFixtureFactory contextFixtureFactory)
[5/8/2018 12:11:21 AM Informational] [xUnit.net 00:00:06.4830827]         test\NuGet.Tests.Apex\NuGet.Client.Tests\Tests\E2E\AbuseLinkTests.cs(13,0): at NuGetClient.Test.Integration.Tests.E2E.AbuseLinkTests..ctor(ProductContextFixtureFactory productContextFixtureFactory)
[5/8/2018 12:11:21 AM Informational] [xUnit.net 00:00:06.5193136]   Finished:    NuGetClient.Test.Integration
